administrator-manual/en/webmail.rst

```diff
@@ -65,4 +65,16 @@ If you want to open the access from any network: ::
 
   config setprop roundcubemail access public
   signal-event nethserver-roundcubemail-update
+  
+Removing
+========
+
+You can remove Roundcube to add another webmail type (eg: SOGo, WebTop.…) and you can keep all the mail server RPMs.
+
+To do this only remove Roundcube from the Installed list in the Software Center.
+
+If you want remove Roundcube, remove all the mail server RPMs and no longer have an email system on the server, first remove Roundcube from the Installed list in the Software Center and when it's completed do this step on the server console – either directly or by ssh. ::
+
+   yum autoremove
```

Comment:
Can we be sure this command does not remove also other dependencies?

I'd change the paragraph to just say:

> do this step on the server console – either directly or by ssh:
     yum autoremove nethserver-roundcubemail